controllers/controller/devworkspacerouting/suite_test.go

```
@@ -109,6 +112,8 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	err = routev1.Install(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
+	err = gwapiv1.AddToScheme(scheme.Scheme)
```

golangci-lint (2.12.2)

[error] SA1019: gwapiv1.AddToScheme is deprecated: use Install instead 

(staticcheck)